spec/features/user_login_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+require 'helpers/user_helpers'
+
+feature 'A user tries to login' do
+
+  before(:each) do
+    User.create(email: 'test@test.com',
+                password: '123',
+                password_confirmation: '123',
+                name: 'John',
+                username: 'test_user1')
+  end
+
+  scenario 'with correct details and is successful' do
```

Comment:
reading this standalone, it isn't clear what the expectation is.  Something like 'succeeds with correct details' might be better.